tests/components/cloud/test_account_link.py

```diff
@@ -41,27 +44,24 @@ def logger(self) -> logging.Logger:
     account_link.async_setup(hass)
 
     with patch(
-        "homeassistant.components.cloud.account_link._get_services",
-        side_effect=lambda _: mock_coro(
-            [
-                {"service": "test", "min_version": "0.1.0"},
-                {"service": "too_new", "min_version": "100.0.0"},
-            ]
-        ),
+            "homeassistant.components.cloud.account_link._get_services",
+            side_effect=lambda _: mock_coro([
+                {
+                    "service": "test",
+                    "min_version": "0.1.0"
+                },
+                {
+                    "service": "too_new",
+                    "min_version": "100.0.0"
+                },
+            ]),
     ):
-        assert (
-            await config_entry_oauth2_flow.async_get_implementations(
-                hass, "non_existing"
-            )
-            == {}
-        )
-        assert (
-            await config_entry_oauth2_flow.async_get_implementations(hass, "too_new")
-            == {}
-        )
+        assert (await config_entry_oauth2_flow.async_get_implementations(
+            hass, "non_existing") == {})
+        assert (await config_entry_oauth2_flow.async_get_implementations(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)